### ISSUE TITLE
Installation fails with upcoming ruby 3.2

### ISSUE THREAD

**Issue by mtasaka** [mvz/alexandria-book-collection-manager#197]
Now Fedora 38 is preparing for upcoming ruby 3.2 (which is expected to be released on 2022/Dec), and now alexandria 0.7.9 fails to install with ruby3.2dev:

```
+ cd alexandria-book-collection-manager-0.7.9
+ rake build --trace
** Invoke build (first_time)
** Invoke autogen (first_time)
** Invoke lib/alexandria/default_preferences.rb (first_time)
** Invoke Rakefile (first_time, not_needed)
** Invoke schemas/alexandria.schemas (first_time, not_needed)
** Execute lib/alexandria/default_preferences.rb
** Execute autogen
** Invoke gettext (first_time)
** Invoke share/locale/cs/LC_MESSAGES/alexandria.mo (first_time)
** Invoke po/cs.po (first_time, not_needed)
** Execute share/locale/cs/LC_MESSAGES/alexandria.mo
rake aborted!
NoMethodError: undefined method `exists?' for FileTest:Module
/builddir/build/BUILD/alexandria-book-collection-manager-0.7.9/util/rake/gettextgenerate.rb:57:in `block in generate_po_files'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:281:in `block in execute'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:281:in `each'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:281:in `execute'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:243:in `block in invoke_prerequisites'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:241:in `each'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:241:in `invoke_prerequisites'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:218:in `block in invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:243:in `block in invoke_prerequisites'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:241:in `each'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:241:in `invoke_prerequisites'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:218:in `block in invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
/usr/share/gems/gems/rake-13.0.6/lib/rake/task.rb:188:in `invoke'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:160:in `invoke_task'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:116:in `block (2 levels) in top_level'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:116:in `each'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:116:in `block in top_level'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:125:in `run_with_threads'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:110:in `top_level'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:83:in `block in run'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
/usr/share/gems/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
/usr/share/gems/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/usr/bin/rake:25:in `load'
/usr/bin/rake:25:in `<main>'
Tasks: TOP => build => gettext => share/locale/cs/LC_MESSAGES/alexandria.mo
```

`FileTest.exists?` is deprecated at least since ruby 2.1, now ruby 3.2 completely removes this method. Replacing with `FileTest.exist?` should fix this.

https://docs.ruby-lang.org/ja/2.1.0/method/File/s/exists=3f.html
https://github.com/ruby/ruby/blob/4954c9fc0f9d06aa4e3e8deb33b41f3fae294adc/NEWS.md?plain=1#L257-L262

**Comment by mvz** [comment]
It looks like this issue was already fixed in master some time ago after complaints from RuboCop. I'll make a new release.

**Comment by mvz** [comment]
I just released version 0.7.10 that should work with Ruby 3.2.
